liquibase-cdi/src/main/java/liquibase/integration/cdi/SchemesCDIConfigBuilder.java

```diff
@@ -190,7 +190,7 @@ CDILiquibaseConfig fileLocked(final String id, Callable<CDILiquibaseConfig> acti
         FileLock lock = null;
         try (
                 FileOutputStream fileStream = new FileOutputStream(lockPath);
-                FileChannel fileChannel = fileStream.getChannel();
+                FileChannel fileChannel = fileStream.getChannel()
```

Comment:
**TIL** ... try-with-resources does not require a `;` in a single resource statement.